CONTRIBUTING.md

```diff
@@ -30,44 +30,44 @@ Since the new GitHub Issue forms we only suggest you to include most information
 
 Unsure where to begin contributing to this project? You can start by looking through these beginner-friendly issues:
 
-- [Beginner issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
-- [Help wanted issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) - issues which should be a bit more involved than `beginner` issues.
+*   [Beginner issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
+*   [Help wanted issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) - issues which should be a bit more involved than `beginner` issues.
 
 ### 📣 Pull Requests
 
 The process described here has several goals:
 
-- Maintain the project's quality
-- Fix problems that are important to users
-- Engage the community in working toward the best possible!
-- Enable a sustainable system for maintainers to review contributions
+*   Maintain the project's quality
+*   Fix problems that are important to users
+*   Engage the community in working toward the best possible!
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Engage the community in working toward the best possible!
```